### DISCUSSION TITLE
How can i get penalty by height? and the reason?

### DISCUSSION THREAD

**Discussion by hsn999** [filecoin-project/lotus#7787] [General Q&A]
How can i get penalty by height?and the reason?

**Comment by Reiers** [comment]
Can you please elaborate on this question? 

Im sending this to Lotus Discussion for help.

**Comment by LexLuthr** [comment]
@hsn999 Can you please elaborate on what you are looking for?

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author for more than 7 days.
